src/native/libs/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -184,7 +184,7 @@ else ()
 endif ()
 
 if((NOT CLR_CMAKE_USE_SYSTEM_ZLIB) AND STATIC_LIBS_ONLY)
-    install_static_library(zlib aotsdk nativeaot)
+    install_static_library(zlib ${STATIC_LIB_DESTINATION} nativeaot)
```

Comment:
Oh I'll have to either split this by OS or make msbuild changes for windows as well.

Reply:
Windows does not use `IlcFrameworkNativePath` as in Unix when the escaping is done, so I'll split this install into a unix branch and another for everything else.

Reply:
Largely you can interpret this as, install so that it's next to lib.system.compression.native.a or the .lib on Windows. It's logically part of the same requirements from a product shipping perspective